bangazonapi/views/profile.py

```diff
@@ -185,7 +185,7 @@ def cart(self, request):
                 cart = {}
                 cart["order"] = OrderSerializer(open_order, many=False, context={
                                                 'request': request}).data
-                cart["order"]["line_items"] = line_items.data
+                cart["order"]["lineitems"] = line_items.data
```

Comment:
I mistakenly approved this line yesterday, but this line should be removed to fix the duplicate line item ticket